types/rss/index.d.ts

```diff
@@ -3,8 +3,7 @@
 // Definitions by: Second Datke <https://github.com/secondwtq>
 // Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
 
-declare var factory: NodeRSS.RSSFactory;
-export = factory;
+export = NodeRSS.RSS;
 
 declare namespace NodeRSS {
```

Comment:
Instead of exporting something in the `NodeRSS` namespace, you should use declaration merging. Otherwise, all the declared interfaces won't be available outside the module.

You should do something like this:

```ts
export = RSS;

declare class RSS {
  ...
}

declare namespace RSS {
  ...
}
```

Note that the `namespace` and the `class` **must** have the same name.

You can test what I mean by "declared interfaces won't be available outside the module" by adding the following snippet to the test code:
```ts
// test type exports
type Types = typeof RSS | RSS | RSS.FeedOptions | RSS.EnclosureObject | RSS.ItemOptions | RSS.XmlOptions;
```

Currently, this will produce errors, with the changes I've proposed this should work.

Reply:
Updated as suggested. Thank you for taking the time to review it!

We ended up rewriting the lib in ts, so the DT types have totally dropped off my priority list.